daemon/api/endpoints/workspaces.py

```diff
@@ -31,11 +31,18 @@ async def _clear_all():
     path='/{id}',
     summary='Deleting an existing Workspace',
 )
-async def _delete(id: DaemonID):
+async def _delete(id: DaemonID, container_only: bool = False):
```

Comment:
The list of supported args here can be `network`, `container`, `files`, `everything`. We'd call `store.delete` for all cases, which would validate before removing.

Reply:
I added the parameters you mentioned, but files/network won't do anything for now. I think we should add this functionality in a separate PR?

container = True -> Delete container
everything = True -> Delete everything including workspace